app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt

```diff
@@ -171,6 +171,17 @@ class AddProfileActivityTest {
     assertThat(label).isEqualTo(context.getString(R.string.add_profile_activity_label))
   }
 
+  @Test
+  fun testAddProfileActivity_required_heading_text_view_hasCorrectRequiredString() {
+    launch(
+      AddProfileActivity::class.java
+    ).use {
```

Comment:
```suggestion
    launch(AddProfileActivity::class.java).use {
```

Nit: this can fit on one line.